sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Scenario.Tests/tests/MultipleStorageAccountsEndToEndTests.cs

```diff
@@ -51,7 +52,7 @@ public async Task BlobToBlob_DifferentAccounts_PrimaryToSecondary_Succeeds()
 
             await TestHelpers.Await(async () =>
             {
-                var pageable = _fixture.OutputContainer2.GetBlobsAsync(prefix: "blob1");
+                var pageable = _fixture.OutputContainer2.GetBlobsAsync(new GetBlobsOptions() { Prefix = "blob1" });
```

Comment:
Curious: why would using the overload with all required parameters not work? That is the change I made in https://github.com/Azure/azure-sdk-for-net/pull/52971/commits/3dd07e6d42f51d0705afd117d7668a9886266bb7

Reply:
I will actually do what you did with the required parameters, because I want to let the webjobs blobs package to still use the package reference, and not have to use the project reference (for the next time we need to release).